packages/css/src/components/blockquote/README.md

```diff
@@ -1,14 +1,14 @@
 # Blockquote
 
-De blockquote (in het Nederlands: citaat) wordt gebruikt om uitspraken te accentueren. Een citaat is anders vormgegeven dan de broodtekst, zodat deze opvalt.
+We use blockquotes to emphasize citations.
+A quote appears differently from the main text to make it stand out.
 
```

Comment:
The line below mentions that the quotes are inserted automatically but that part is not being translated.